screens/HotspotDetailScreen.tsx

```diff
@@ -230,92 +252,72 @@ export const HotspotDetailScreen = ({
-                size={24}
-                icon="check-circle"
-                color={theme.colors.text}
-                style={styles.catCategoryIcon}
-              />
-              <Title style={styles.catCategoryTitleLabel}>
-                Pisici sterilizate
-              </Title>
+          <View
+            style={{
+              alignItems: 'center',
+            }}
+          >
+            <View style={styles.contentCotainer}>
+              <View style={styles.addressContainer}>
+                <Title style={styles.addressTitle}>{address}</Title>
+                <Button
+                  style={styles.editButton}
+                  contentStyle={styles.buttonContent}
+                  labelStyle={styles.buttonLabel}
+                  icon="pencil-outline"
+                  mode="contained"
+                  onPress={() => {
+                    navigation.navigate('AddHotspot', {
+                      region: region!,
+                      isUpdate: true,
+                    });
+                  }}
+                  compact
+                >
+                  Editează
+                </Button>
+              </View>
+              <Caption style={styles.subTitle}>
+                {hotspotDetails.description}
+              </Caption>
+              <View style={styles.separator} />
+              {!!hotspotDetails.notes && (
+                <Caption style={styles.notes}>{hotspotDetails.notes}</Caption>
+              )}
+              <SummaryView hotspotDetails={hotspotDetails} />
+              <View style={styles.catCategoryContainer}>
+                <IconButton
+                  size={24}
+                  icon="close-circle"
+                  color={theme.colors.text}
+                  style={styles.catCategoryIcon}
+                />
+                <Title style={styles.catCategoryTitleLabel}>
+                  Pisici nesterilizate
+                </Title>
+              </View>
+              <CatsView cats={hotspotDetails.unsterilizedCats} />
+              <View style={styles.separator} />
+              <View style={styles.catCategoryContainer}>
+                <IconButton
+                  size={24}
+                  icon="check-circle"
+                  color={theme.colors.text}
+                  style={styles.catCategoryIcon}
+                />
+                <Title style={styles.catCategoryTitleLabel}>
+                  Pisici sterilizate
+                </Title>
+              </View>
+              <CatsView cats={hotspotDetails.sterilizedCats} />
             </View>
-            <CatsView cats={hotspotDetails.sterilizedCats} />
           </View>
+
           <View style={styles.imageView}>
             <Image
-              source={catLady3}
+              source={Platform.OS === 'web' ? catLady3Web : catLady3}
               style={styles.image}
-              resizeMode="contain"
+              resizeMode={Platform.OS === 'web' ? 'cover' : 'contain'}
```

Comment:
This is not good, we should decide based on screen/window width, not the platform, because the user can use the web app on a mobile device.
Also, this kind of logic needs to be separated from the business logic, i.e. extract a `isWideScreen` function in a "tools" module, then as a second step, this footer can be extracted to a separate component.